crAssphage Assembly Workshop
============================

Keywords
--------

Alignment, Differential-expression, Feature-summarisation, Pre-processing, QC

Authors
-------

Bas Dutilh, http://tbb.bio.uu.nl/dutilh

Type
----

-  Course

Description
-----------

In this course we will assemble viral metagenomes derived from twelve human fecal samples, using de novo cross-assembly. We will use depth profiling to discover sequence elements (contigs) that are shared between the metagenomes and identify them. The aim is to get acquainted with metagenome assembly and the concept of binning by using depth profiles.

Aims
----



Prerequisites
-------------
Some experience with command line will be useful, although links are provided with explanation of the linux commands

Target audience
---------------
Wet lab biologists with little programming experience.

Learning objectives
-------------------

- Perform a simple metagenome cross-assembly
- Identify contigs present in many samples
- Identify contigs with correlating depth profiles
- Students who are less familiar with the command line will also be able to pick up some linux basics


Materials
---------
Two one-hour preparatory lectures can be given in the morning and can be found here: 
* `CABBIO <http://www.slideshare.net/bedutilh/metagenome-sequence-assembly-cabbio-20150629-buenos-aires>`_ 
* `VIRAL <http://www.slideshare.net/bedutilh/viral-metagenomics-cabbio-20150629-buenos-aires>`_

The hands-on workflow can be found at `here <http://tbb.bio.uu.nl/dutilh/CABBIO/>_ and should be doable in one afternoon with supervision. There is also an iPython notebook about this `here: <https://github.com/linsalrob/CrAPy>`_

Data
----
The data consists of twelve viral metagenomes derived from human fecal samples from different individuals (described in PMID 20631792) and can be downloaded here: http://www.ncbi.nlm.nih.gov/sra/SRX023429


Timing
------

1 day

Content stability
-----------------

Stable

Technical requirements
----------------------
idba_hybrid, bowtie2, crAss v2.0, SPAdes v3.5.0

Literature references
---------------------

-  to do

Changelog
---------
