==========================
List of Training Materials
==========================

Background Material
===================

Factors influencing course design
---------------------------------
* Organisms
* Audience/Student expertise
* Course length
* Compute resources 
* Trainer knowledge
* Environment/Biome
* Available Datasets

Links to resources that cover several for the above topics
----------------------------------------------------------
* `CABBIO <http://tbb.bio.uu.nl/dutilh/CABBIO/>`_
* `EDAMAME <www.edamamecourse.org>`_
* `de.NBI/DZIF Workshop <http://dzif-metagenomics-workshop.readthedocs.org/en/latest/>`_
* `CAMI Challenge <https://data.cami-challenge.org/>`_

Background
----------
* Experimental Design
* Computer Literacy
* Cloud Computing
* Sequencing options
* Statistical power

Links to resources that cover several for the above topics
----------------------------------------------------------
* `Morgan Langille <http://bioinformatics.ca/metagenomics2015module1pptx>`_
* `TGAC <https://github.com/TGAC/361Division/blob/master/Metagenomics%202015/alston_TGAC-metagenomics-from-bench-to-data-analysis_FINAL.pptx>`_
* `VanHijum tutorial <http://www.nbic.nl/uploads/media/Day1_vanHijum_introduction_metagenomics_2013.pdf>`_
* `Linux from BAS <http://bioinformatics.bio.uu.nl/manuals/introlinux/>`_
* `Linux from TGAC <https://github.com/TGAC/361Division/blob/master/Metagenomics%202015/Introduction%20to%20Linux.docx>`_
* `UC Davis Korf Lab <http://korflab.ucdavis.edu/bootcamp.html>`_
* `EDAMAME Shell <https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22-introduction_to_the_shell.md>`_
* `Software Carpentry Shell <http://swcarpentry.github.io/shell-novice/>`_
* `Python <https://github.com/datacarpentry/python-ecology-lesson>`_
* `Github <http://swcarpentry.github.io/git-novice/>`_
* `Data Carpentry Automation <https://github.com/qingpeng/2016-02-22-isu/blob/gh-pages/lessons/Day1Afternoon-Shell_by_Jin/genomic_shell_04_automating_a_workflow.md>`_
* `EDAMAME Cloud Computing <https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22-EC2_Startup.md>`_

Pre-Processing
==============

Data treatment/pre-processing
-----------------------------
* Quality
* Filtering Reads
* Trimming Reads
* Contamination
* Barcodes / multiplexing
* Read Merging

Links to resources that cover several for the above topics
----------------------------------------------------------
* `Methods for Microbial Ecologists <http://metagenomic-methods-for-microbial-ecologists.readthedocs.org/en/latest/day-4/>`_
* `EDAMAME Read Processing <https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-22_FastQC_tutorial.md>`_
* `BPA CSIRO <https://github.com/BPA-CSIRO-Workshops/btp-module-ngs-qc>`_
* `Metagenomics TGAC <https://github.com/TGAC/361Division/blob/master/Metagenomics%202015/Metagenomics%20v4.pdf>`_

Assembly
--------
* Short reads
* Long reads
* Hybrid
* Data reduction / Clustering / Dereplication

Links to resources that cover several for the above topics
----------------------------------------------------------
* `Dutilh Assembly <http://www.nbic.nl/uploads/media/Day3_Dutilh_Metagenome_assembly_2013.pdf>`_
* `Assembly <https://drive.google.com/open?id=0B-b6Xw3NFeyKWjIyTFVtUEhuSkk>`_
* `EDAMAME workshop <https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-25-assembling_your_metagenome.md>`_

Types of Analysis
=================

Marker gene amplicons
---------------------
* 16s rRNA, 18s rRNA, ITS1/2
* Taxonomic profiling
* OTU clustering 
* taxonomy -> function inferences

Links to resources that cover several for the above topics
----------------------------------------------------------
* `Hsiao tutorials <http://bioinformatics.ca/metagenomics2015module2pptx>`_
* `Morgan Langille <https://www.dropbox.com/s/e8e6p62k92s1fo9/Langille_PICRUSt_STAMP_Lecture.pptx?dl=1>`_
* `EDAMAME course <www.edamamecourse.org>`_

Shotgun metagenomics & metatranscriptomics
------------------------------------------
* To assemble or not?
* To gene call or not? 
* Kmers (“reference-free”)
* Taxonomic profiling
* Coverage estimation
* Reference mapping / alignment
* Gene Catalogues
* Marker-Based approaches
* Read-based approaches
* Functional profiling/annotation
* Functional hierarchies / ontologies	
* Depth comparisons
* Binning
* Whole genome assembly and evaluation
* Pathway Analysis / Metabolic modeling
* Cross assembly

Links to resources that cover several for the above topics
----------------------------------------------------------
* `John Parkinson <http://bioinformatics.ca/metagenomics2015module5pptx>`_
* `Reference Tutorial <http://www.nbic.nl/uploads/media/Day2_deJager_MG_RAST_workshop_2013.pdf http://www.nbic.nl/uploads/media/MGRast_workshop_130207.pdf>`_
* `BioMAS <https://www.dropbox.com/s/ijwkd4ifgb9nlb2/BioMaS_demo_Recas.mov?dl=0>`_
* `Morgan Langille metagenomic tutorial <http://bioinformatics.ca/metagenomics2015module3pptx>`_
* `Metagenomics Crash Course <http://metag-crash-course.readthedocs.org/en/latest/>`_
* `CABBIO <http://tbb.bio.uu.nl/dutilh/CABBIO/>`_
* `TGAC <https://github.com/TGAC/361Division/blob/master/Metagenomics%202015/TGAC_metagenomics_151015_publicdataresources_JP.pdf>`_

Comparative Analyses
====================

Statistics and visualization
----------------------------
* Metadata integration
* Sample comparison
* Ecological measurements/indexes
* Multivariate and comparative statistics
* Machine Learning

Links to resources that cover several for the above topics
----------------------------------------------------------
* `STAMP <https://www.dropbox.com/s/e8e6p62k92s1fo9/>`_
* `Dutilh <http://www.nbic.nl/uploads/media/Day3_Dutilh_Comparative_metagenomics_2013.pdf>`_
* `Luis Pedro <http://mybinder.org/repo/luispedro/StructureFunctionOceanTutorial>`_
* `Stuart Jones EDAMAME <https://github.com/edamame-course/2015-tutorials/blob/master/final/2015-06-27-Jones_R.md>`_
* `BPA CSIRO <https://github.com/BPA-CSIRO-Workshops/metagenomics-module-vis/blob/master/handout/vis.tex>`_








